packages/manager/modules/pci/src/projects/project/project.service.js

```diff
@@ -20,38 +22,71 @@ export default class {
     return this.project;
   }
 
-  isManuallyReviewedByAntiFraud() {
+  getProjectOrderStatus(projectId) {
+    const { VALIDATING } = ORDER_FOLLOW_UP_STEP_ENUM;
+    const {
+      FRAUD_MANUAL_REVIEW,
+      FRAUD_REFUSED,
+      PAYMENT_INITIATED,
+      PAYMENT_RECEIVED,
+    } = ORDER_FOLLOW_UP_HISTORY_STATUS_ENUM;
+
+    const status = {
+      orderId: null,
+      isActivating: false,
+      isManuallyReviewedByAntiFraud: false,
+      isRefusedByAntiFraud: false,
+      voucher: '',
+    };
+
     return this.$http
-      .get(`/cloud/order/?planCode=project.2018`)
+      .get(`/cloud/order/?planCode=${FULL_PROJECT_PLANCODE}`)
       .then(({ data: orders }) => {
-        const validatingOrder = orders.find(
-          (order) =>
-            order.status.toLowerCase() ===
-            ORDER_FOLLOW_UP_STEP_ENUM.DELIVERING.toLowerCase(),
-        );
-
-        return validatingOrder
-          ? this.getOrderFollowUp(validatingOrder?.orderId)
-          : false;
-      })
-      .then((orderFollowUp) => {
-        if (orderFollowUp) {
-          const deliveringStep = orderFollowUp?.find(
-            (element) => element.step === ORDER_FOLLOW_UP_STEP_ENUM.VALIDATING,
-          );
-          return (
-            deliveringStep?.history.find(
-              (event) =>
-                event.label ===
-                ORDER_FOLLOW_UP_HISTORY_STATUS_ENUM.FRAUD_MANUAL_REVIEW,
-            ) || false
-          );
+        const order = orders
+          .sort(({ date: dateA }, { date: dateB }) => (dateB > dateA ? 1 : -1))
+          .find(({ serviceName }) => serviceName === projectId);
```

Comment:
**question:** What is the need for `sorting` if it is just to find the correct project order?